.github/workflows/deploy-api-v2.yml

```diff
@@ -0,0 +1,20 @@
+name: Deploy HTTP API
+on:
+  push:
+    branches:
+      - beta
+
+jobs:
+  deploy:
+    name: Staging API Deploy
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+      - name: Add Heroku remote
+        run: heroku git:remote --app $HEROKU_APP_NAME
+        env:
+          HEROKU_APP_NAME: ${{ secrets.HEROKU_APP_NAME }}
+      - name: Push to Heroku
+        run: git push heroku -f `git subtree split --prefix ironfish-http-api-v2 HEAD`:master
```

Comment:
How is this working? Doesn't this require that you build it and push up the built files? or does the node buildpack now automatically install and build?

Reply:
This pushes the sub-folder for the new API, and the `heroku/nodejs` buildpack installs dependencies / runs the build

Reply:
nice, that's new since I last used it. We used to have to use multibuildpack with custom buildpack and do the build ourselves.